src/Client.cpp

```diff
@@ -882,11 +878,8 @@ void CClient::HandleCap(const CString& sLine)
 	if (sSubCmd.Equals("LS")) {
 		SCString ssOfferCaps;
 		GLOBALMODULECALL(OnClientCapLs(this, ssOfferCaps), NOTHING);
-		CString sRes;
-		for (SCString::iterator i = ssOfferCaps.begin(); i != ssOfferCaps.end(); ++i) {
-			sRes += *i + " ";
-		}
-		RespondCap("LS :" + sRes + "userhost-in-names multi-prefix znc.in/server-time-iso znc.in/batch znc.in/self-message");
+		CString sRes = CString(" ").Join(ssOfferCaps.begin(), ssOfferCaps.end());
+		RespondCap("LS :" + sRes + " userhost-in-names multi-prefix znc.in/server-time-iso znc.in/batch znc.in/self-message");
```

Comment:
If no module adds any cap, this will add an extra space in the beginning.
Let's add these strings to the set before `Join`ing